rudderstack/integrations/destinations/destination_customerio.go

```diff
@@ -46,10 +46,10 @@ func init() {
 			Description:      "Enter the name of the event that is fired immediately after setting the device token.",
 			ValidateDiagFunc: c.StringMatchesRegexp("(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{1,100})$"),
 		},
-		"datacenter_eu": {
-			Type:        schema.TypeBool,
-			Optional:    true,
-			Description: "Enable this option in case your account is based in the EU region.",
+		"datacenter": {
+			Type:        schema.TypeString,
+			Optional:    false,
+			Description: "Select your Customer.io Data Center.",
```

Comment:
Shouldn't it be "Input your Customer.io ..." since it is a text field?

Reply:
Update the desription, thanks